### DISCUSSION TITLE
Is it possible to ingest additional features (i.e. numerical, categorical, sensor inputs) alongside Images?

### DISCUSSION THREAD

**Discussion by jabbasj** [open-edge-platform/anomalib#1272] [Feature Requests]
I want to augment my dataset with additional sensor data, is this possible & if so can you please point me in the right direction?
If not, what would you recommend?

**Comment by samet-akcay** [comment]
Hi @jabbasj, can you elaborate a bit more how exactly you want to augment your dataset? I'm not sure whether you want to add this additional data as the 4th dimension to the RGB data, or you want to add on top of the existing RGB images.

**Comment by jabbasj** [comment]
> Hi @jabbasj, can you elaborate a bit more how exactly you want to augment your dataset? I'm not sure whether you want to add this additional data as the 4th dimension to the RGB data, or you want to add on top of the existing RGB images.

The data I am looking to include is not really per-pixel... I want to associate many things like weather conditions, pressure readings, temperature readings, type of object in the image, etc...

**Comment by samet-akcay** [comment]
Multi-modality is something we would like to work on, but unfortunately we do not have resources (time etc) to work on it yet.
